applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -42,11 +36,8 @@ def setUp(self):
         # Set to true to get post-process files for the test
         self.print_output = False
 
+    @UnitTest.skipIfApplicationsNotAvailable("ExternalSolversApplication", "HDF5Application")
```

Comment:
Can we also start replacing the ExternalSolvers used in the test with the LinearSolvers? Or is it better to wait?

Reply:
yes!
no there will be no more interface changes